hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFutureTest.java

```diff
@@ -36,91 +36,52 @@ public void setup() {
         operationService = getOperationService(local);
     }
 
-    // ================= isDone ==========================================
-
     @Test
     public void isDone_whenNullResponse() throws ExecutionException, InterruptedException {
         DummyOperation op = new DummyOperation(null);
 
         InternalCompletableFuture future = operationService.invokeOnTarget(null, op, getAddress(local));
-        // first we wait for the future to complete.
-        future.get();
+        future.get(); // first we wait for the future to complete.
```

Comment:
It's a test - so no checkstyle.
In a test I personally prefer this notation to make the test smaller in terms of LOC.
I have no problems in moving it up again.